test/worlds.jl

```diff
@@ -305,8 +305,6 @@ src4 = code_typed(applyf35855_2, (Vector{Any},))[1]
 @test !(wany4 == wany3) || equal(src4, src3) # code doesn't change unless you invalidate
 
 ## ambiguities do not trigger invalidation
-using Printf
-Printf.gen("%f")
```

Comment:
@timholy, can you comment on the use of `Printf.gen` here? Is there another method we can call to get similar results? I'd like to get rid of `Printf.gen` in the rewrite, so any suggestion to change this would be great.

Reply:
Sorry I missed this ping. It might have been stale, it doesn't look coupled to anything else.

I hadn't followed this, but it looks like awesome work. Many thanks!